e2e/process-services/start-process-component.e2e.ts

```diff
@@ -75,6 +75,7 @@ describe('Start Process Component', () => {
     describe('Provider: BPM', () => {
 
         beforeAll(async () => {
+            try {
```

Comment:
Please fix the formatting. You can do it with IntelliJ by selecting the text and press `option + command + L`

Reply:
done